src/membership/members.rs

```diff
@@ -122,14 +150,16 @@ decl_storage! {
         /// MemberId to assign to next member that is added to the registry
         pub NextMemberId get(next_member_id) build(|config: &GenesisConfig<T>| config.first_member_id): T::MemberId = T::MemberId::from(DEFAULT_FIRST_MEMBER_ID);
 
-        /// Mapping of member ids to their corresponding primary accountid
+        /// Mapping of member ids to their corresponding primary account_id
         pub AccountIdByMemberId get(account_id_by_member_id) : map T::MemberId => T::AccountId;
 
-        /// Mapping of members' account ids to their member id.
+        /// Mapping of members' primary account ids to their member id.
         pub MemberIdByAccountId get(member_id_by_account_id) : map T::AccountId => Option<T::MemberId>;
 
+        /// Mapping of members' controller account ids to their member id.
+        pub MemberIdByControllerAccountId get(member_id_by_controller_account_id) : map T::AccountId => Option<T::MemberId>;
```

Comment:
I don't think we need this, and we also should not _require_ that controller accounts are unique across members anyway.

Reply:
Obviously this is discussed more deeply in the PR comment, but its there to make lookup of a member by their controller account faster. 

If we don't want to enforce uniqueness of the key then it can be a `map T::AccountId => Vec<T::MemberId>`